src/main/kotlin/no/nav/dagpenger/streams/River.kt

```diff
@@ -14,10 +15,23 @@ abstract class River : Service() {
     override fun buildTopology(): Topology {
         val builder = StreamsBuilder()
         val stream = builder.consumeTopic(DAGPENGER_BEHOV_PACKET_EVENT)
-        stream.peek { key, packet -> LOGGER.info("Processing $packet with key $key") }
+        stream
```

Comment:
Should you care if this is a packet you are not interested in?

Reply:
no, good point. Will remove it - already log an error on runCatching